Add `Community Mentorship Program Winter 2024 Signups Now Open` news post (#12235)
ppy/osu-wiki: news/2024/2024-09-29-community-mentorship-program-autumn-2024-signups-now-open.md

---
layout: post
title: Community Mentorship Program Winter 2024 Signups Now Open
date: 2024-09-29 10:00:00 +0000
---

> The 28th cycle of the Community Mentorship program is upon us! Get ready to delve deep into the world of mapping, modding, or storyboarding.

Reviewer: ```suggestion
The 28th cycle of the Community Mentorship Program is upon us! Get ready to delve deep into the world of mapping, modding, or storyboarding.
```

![](/wiki/shared/news/banners/community-mentorship-program.jpg)

> The Community Mentorship program serves as a place for upcoming mappers, modders, and storyboarders to develop their skills under the guidance of more experienced members in the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.

Reviewer: ```suggestion
The Community Mentorship Program serves as a place for upcoming mappers, modders, and storyboarders to develop their skills under the guidance of more experienced members in the community. If you've ever found yourself wanting to expand your current knowledge, this might just be the opportunity you need.
```

> Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events such as feedback lobbies and speedmapping contests, organise guest difficulties for your mapsets on our job board, and more! General mapping discussion, feedback channels and the guest difficulty job board are now open for anyone to contribute, so if this sounds interesting to you, [join our Discord server to take part](https://discord.gg/Ft2FtXmBgx).

Reviewer: ```suggestion
Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events such as feedback lobbies and speedmapping contests, organize guest difficulties for your mapsets on our job board, and more! General mapping discussion, feedback channels and the guest difficulty job board are now open for anyone to contribute, so if this sounds interesting to you, [join our Discord server to take part](https://discord.gg/Ft2FtXmBgx).
```

Since Program is american english it makes sense to use the american word organize vs the british organise.

Reviewer 2: "program" is just part of a name here. it wouldn't be changed to "Community Mentorship Programme" just because the article is in british english

> Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events such as feedback lobbies and speedmapping contests, organize guest difficulties for your mapsets on our job board, and more! General mapping discussion, feedback channels and the guest difficulty job board are now open for anyone to contribute, so if this sounds interesting to you, [join our Discord server to take part](https://discord.gg/Ft2FtXmBgx).

Reviewer 2: ```suggestion
Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events such as feedback lobbies and speedmapping contests, organise guest difficulties for your mapsets on our job board, and more! General mapping discussion, feedback channels and the guest difficulty job board are now open for anyone to contribute, so if this sounds interesting to you, [join our Discord server to take part](https://discord.com/invite/Ft2FtXmBgx).
```

If you're experienced in mapping, modding, or storyboarding, and have a desire to take those who are new under your wing, then consider joining us as a mentor! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/1983459?n=1), you'll be able to apply right away. If you're only just getting started and are eager to hone your skills, then signing up as a mentee might be what you need!

---

For this following cycle, the timeline is as follows (all opening and closing dates start and end at midnight UTC+0):

- Mentor signups will open on **September 29th** and close on **October 6th**.
- Mentee signups will open on **October 20th** and close on **October 27th**.
- The 2024 Winter cycle will run from **October 27th** to **February 9th**.

> Throughout the cycle, the organization team will host a speed-mapping contest for mentees to participate in. In the past, we've also ran feedback lobbies and other types of mapping contests, created [mapping puzzles](https://osu.ppy.sh/community/forums/topics/565382), and hosted [mapping discussion panels](https://osu.ppy.sh/community/forums/topics/514122). If you are a mentor or outside party interested in running or trialing a new event on the server, feel free to reach out to us!

Reviewer: ```suggestion
Throughout the cycle, the organization team will host a speed-mapping contest for mentees to participate in. In the past, we've also run feedback lobbies and other types of mapping contests, created [mapping puzzles](https://osu.ppy.sh/community/forums/topics/565382), and hosted [mapping discussion panels](https://osu.ppy.sh/community/forums/topics/514122). If you are a mentor or outside party interested in running or trialing a new event on the server, feel free to reach out to us!
```

> Throughout the cycle, the organization team will host a speed-mapping contest for mentees to participate in. In the past, we've also run feedback lobbies and other types of mapping contests, created [mapping puzzles](https://osu.ppy.sh/community/forums/topics/565382), and hosted [mapping discussion panels](https://osu.ppy.sh/community/forums/topics/514122). If you are a mentor or outside party interested in running or trialing a new event on the server, feel free to reach out to us!

Reviewer 2: ```suggestion
Throughout the cycle, the organisation team will host a speed-mapping contest for mentees to participate in. In the past, we've also run feedback lobbies and other types of mapping contests, created [mapping puzzles](https://osu.ppy.sh/community/forums/topics/565382), and hosted [mapping discussion panels](https://osu.ppy.sh/community/forums/topics/514122). If you are a mentor or outside party interested in running or trialling a new event on the server, feel free to reach out to us!
```

> If you or a friend are curious about the rules and which mentors you can apply for, please go to our [forum announcement and rules thread](https://osu.ppy.sh/community/forums/topics/1983459?n=1) for more information.

Reviewer: ```suggestion
If you or a friend is curious about the rules and which mentors you can apply for, please go to our [forum announcement and rules thread](https://osu.ppy.sh/community/forums/topics/1983459?n=1) for more information.
```

The verb talks about the word friend so it should be is instead of are.

> Otherwise: [join our Discord server!](https://discord.gg/Ft2FtXmBgx)

Reviewer 2: ```suggestion
Otherwise: [join our Discord server!](https://discord.com/invite/Ft2FtXmBgx)
```

We're looking forward to the mappers, modders and storyboarders of tomorrow!

> —the Mentorship Organisation Team

Reviewer 3: ```suggestion
—the Mentorship Organization Team
```

Reviewer 3: Since Wala mentioned it should be American instead of British, i think it would make sense aswell, right?

> —the Mentorship Organization Team

Reviewer 2: ```suggestion
—the Mentorship Organisation Team
```
